operator/es_client.go

```diff
@@ -365,8 +365,13 @@ func (c *ESClient) waitForEmptyEsNode(ctx context.Context, pod *v1.Pod) error {
 				default:
 					// Process response as normal if context is not done.
 					var shards []ESShard
-					err := json.Unmarshal(r.Body(), &shards)
+					body := r.Body()
+					c.logger().Debugf("Request to %s/_cat/shards?h=index,ip&format=json status code %d - %s  on %s/%s (%s)",
+						c.Endpoint.String(), r.StatusCode(), string(body[:]), pod.Namespace, pod.Name, podIP)
```

Comment:
We can decide if we want to keep the request body logging only for err != nil cases. One negative right now is the string conversion on every call. Not a risk necessarily.